### PR TITLE
remove argument links to criterion

### PR DESCRIPTION
**Pull request by sfluegel05**
In #69, I added links between `chebi_version` arguments. This way, you only need to type in the chebi version once and avoid the potential for e.g. the semantic loss checking for inconsistencies in version 200, while the training data is from version 231.

However, this leads to issues when someone is using a different loss function than `BCEWeighted`. Then, the link leads to an argument of the loss function that does not exist, causing an error in `jsonargparse` (the error is shown as an `IndexError`, but that is the error it raises while trying to generate an error message, not the actual error).

For now, I removed the argument linking.

### CODE REVIEW

**Comment by sfluegel05** [comment]
@schnamo Could you please check if this solves your problem with the chebai dev branch?

**Comment by schnamo** [comment]
Thank you very much, Simon. This does solve the problem I had with the chebai dev branch.

**Review by schnamo — APPROVED** [review]
Thank you very much, Simon. This does solve the problem I had with the chebai dev branch.
